### DISCUSSION TITLE
Configuration that allows despawn of mobs holding items

### DISCUSSION THREAD

**Discussion by Potothingi** [PaperMC/Paper#10592] [Ideas]
### Is your feature request related to a problem?

Mobs that do not despawn will cause lag.

### Describe the solution you'd like.

Configuration that allows despawn of mobs holding items
+ Blacklist/whitelist of items in hand(rotten_flesh, bone, arrow, egg, ink_sac...)

### Describe alternatives you've considered.

I looked up the configuration.

**Comment by Machine-Maker** [comment]
I think I would prefer some API in the EntityPickupItemEvent to control whether persistence is set for the mob or not rather than a global configuration. That way plugins could control this behavior more specifically.
